### PR TITLE
CORE-7120: Delete Packaging Format Version 1

### PR DESCRIPTION
**Pull request by josephzunigadaly**
- Delete support for Packaging Format Version 1.
- Change test `cannot upload CPI without group policy file aka CPB` to `cannot upload CPB instead of CPI`
   - The original name of this test could lead us down two paths, upload a CPB or upload a CPI with no group policy
   - I tried creating a CPI with no group policy but it turned out to be difficult to do
   - So uploading a CPB instead of a CPI is the easier mistake to make and is more likely to happen
- Updated `CreateCpiTest` to create version 2 CPKs and CPBs

### CODE REVIEW

**Review by driessamyn — APPROVED** [review]
LGTM

File: libs/packaging/packaging/src/main/kotlin/net/corda/libs/packaging/internal/v2/SignatureCollector.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package net.corda.libs.packaging.internal.v2
+
+import java.util.jar.JarEntry
+
+internal class SignatureCollector {
```
Comment (kyriathar) [inline libs/packaging/packaging/src/main/kotlin/net/corda/libs/packaging/internal/v2/SignatureCollector.kt:5]: I think this no longer does what its name suggests, maybe we could be moving its statics in `CpkLoaderV2` where it is being used?

File: tools/plugins/package/src/test/kotlin/net/corda/cli/plugins/packaging/CreateCpiTest.kt [hunk 2]
```diff
@@ -1,28 +1,31 @@
 package net.corda.cli.plugins.packaging
```
Comment (kyriathar) [inline tools/plugins/package/src/test/kotlin/net/corda/cli/plugins/packaging/CreateCpiTest.kt:1]: I think `CreateCpiTest` should be removed? It was there to test `CreateCpi` (V1)? We have `CreateCpiV2Test` to cover `CreateCpiV2`.

File: tools/plugins/package/src/test/kotlin/net/corda/cli/plugins/packaging/CreateCpiTest.kt [hunk 3]
```diff
@@ -156,56 +186,56 @@ class CreateCpiTest {
...
+                .flatMap { cpiEntry ->
+                    if (cpiEntry.name.endsWith(".cpb")) {
+                        // This entry is a CPB. Read the filenames inside.
+                        //
+                        // We do not close the CPB JarInputStream because it would close the CPI JarInputStream and we
+                        // need that open to continue reading the CPI.
+                        readCpbFilenames(cpi, cpiEntry)
+                    } else {
+                        sequenceOf(cpiEntry.name)
+                    }
```
Comment (kyriathar) [inline tools/plugins/package/src/test/kotlin/net/corda/cli/plugins/packaging/CreateCpiTest.kt:221]: Maybe could do without sequences here, hopefully to make it bit simpler?

**Review by kyriathar — COMMENTED** [review]
Late review. Some comments.

**Pull request merged**